FORM_CONVERSION_CHECKLIST.md

```
@@ -0,0 +1,225 @@
+
+#### Exercise Management
+
+- [ ] `apps/frontend/app/routes/_dashboard.fitness.exercises.item.$id._index.tsx`
+  - **State**: Uses `useState` for changingExerciseSettings
+  - **Action**: Convert modal forms to `useForm`
+
+---
+
+## Conversion Priority
+
+### High Priority (Quick Wins)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)